dashboard/api.py

```diff
@@ -27,33 +27,23 @@
 REFRESH_ENROLLMENT_CACHE_MINUTES = 5
 
 
-class CourseStatus:
+class RunStatus:
     """
-    Possible statuses for a course for a user. These are the course run statuses used in the dashboard API.
+    Possible statuses for a course run for a user
     """
+    CURRENTLY_ENROLLED = 'currently-enrolled'
     PASSED = 'passed'
+    WILL_ATTEND = 'will-attend'
+    CAN_UPGRADE = 'can-upgrade'
     NOT_PASSED = 'not-passed'
-    CURRENT_GRADE = 'verified'
-    UPGRADE = 'enrolled'
+    NOT_ACTIONABLE = 'not-actionable'
     OFFERED = 'offered'
 
     @classmethod
     def all_statuses(cls):
-        """Helper to get all the statuses"""
-        return [cls.PASSED, cls.NOT_PASSED, cls.CURRENT_GRADE,
-                cls.UPGRADE, cls.OFFERED]
-
-
-class CourseRunStatus:
-    """
-    Possible statuses for a course run for a user. These are used internally.
-    """
-    NOT_ENROLLED = 'not-enrolled'
-    GRADE = 'grade'
-    READ_CERT = 'read-cert'
-    WILL_ATTEND = 'will-attend'
-    UPGRADE = 'upgrade'
-    NOT_PASSED = 'not-passed'
+        """Helper to get all status values"""
+        return {cls.CURRENTLY_ENROLLED, cls.PASSED, cls.WILL_ATTEND, cls.CAN_UPGRADE,
+                cls.NOT_PASSED, cls.NOT_ACTIONABLE, cls.OFFERED}
```

Comment:
i don't think that is any different from what we currently have. if we changed a value in the old `CourseStatus`, we would need to make the same change in `static/js/constants.js` (`STATUS_PASSED`, `STATUS_NOT_PASSED`, etc). or are you talking about a different kind of change?